Developer-Guide.md

```diff
@@ -63,15 +68,29 @@ $ sudo kata-runtime kata-check
 
 If your system is *not* able to run Kata Containers, the previous command will error and explain why.
 
-## Disable initrd
+## Configure to use initrd or rootfs image
 
-Unless you use an initial ramdisk (`initrd` or `initramfs`):
+Kata containers can run with either an initrd image or a rootfs image.
+
+If you want to test with `initrd`, make sure you have `initrd = /usr/share/kata-containers/kata-containers-initrd.img`
+in `/usr/share/defaults/kata-containers/configuration.toml` and remove the `image` line with e.g.,
 
 ```
-$ sudo sed -i 's/^\(initrd.*\)/# \1/g' /usr/share/defaults/kata-containers/configuration.toml
+$ sudo sed -i 's/^\(image =.*\)/# \1/g' /usr/share/defaults/kata-containers/configuration.toml
 ```
+The initrd image can be created as shown in the [create an initrd image](#create-an-initrd-image---optional) secion.
+
+OTOH, if you want to test with a rootfs `image`, make sure you have `image = /usr/share/kata-containers/kata-containers.img`
```

Comment:
Lines 83-84 suggested rewrite: If you test with a rootfs `image`, make sure you have `image = /usr/share/kata-containers/kata-containers.img` in `/usr/share/defaults/kata-containers/configuration.toml` and remove the `initrd` line with the following: